### PR TITLE
C sharp types

### PR DESCRIPTION
**Pull request by VishnuKulakkattu**
Added Slides for C# Types

### CODE REVIEW

**Review by sneha-brainchild — DISMISSED** [review]

File: CS056AID Types.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                        </li>
+                        <li>
+                            <p>Reference types</p>
+                        </li>
+                    </ul>
+                    <p>Value types include simple types (such as int, float, bool, and char), enum types, struct types, and Nullable value types.</p>
+                    <p>Reference types include class types, interface types, delegate types, and array types.</p>
+                </section>
+                <section>
+                    <img src="/image/CSharpTypes.png">
```
Comment (wandercode) [inline CS056AID Types.html:38]: image cant seen

File: CS056AID Types.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+     
+                    //Output: 
+                        100
+                        200
+                        100
+                </code></pre>
+                </section>
+                <section>
+                    <p>This is how the value type getting stored.</p>
+                    <img src="/image/Typevaluestore.png" alt="value type getting stored">
```
Comment (wandercode) [inline CS056AID Types.html:80]: image cant seen.
and please include this image under second slide. i think it suitable there.

**Review by wandercode — CHANGES_REQUESTED** [review]

File: CS056AID Types.html (hunk 2 above)
Comment (VishnuKulakkattu) [reply CS056AID Types.html:80]: Kindly check and review

**Review by wandercode — APPROVED** [review]

**Comment by wandercode** [comment]
LGTM 👍

**Pull request merged**